docker-compose.yaml

```diff
@@ -17,11 +17,14 @@ services:
       - postgresql
     environment:
       # Basic Configuration
+      EDC_CONNECTOR_NAME: 'example-connector'
```

Comment:
please add hint to use no white spaces etc.

Reply:
We might have made the mistake to use spaces in variables besides EDC_IDS_TITLE and EDC_IDS_DESCRIPTION before, but I think the example value is now a pretty strong hint on this variable being expected not to use spaces. An additional comment line would be a bit too much, imho